examples/poetry.lock

```diff
@@ -0,0 +1,462 @@
+# This file is automatically @generated by Poetry 2.1.1 and should not be changed by hand.
```

Comment:
https://github.com/ni/grpc-device/actions/runs/13816508402/job/38650980325?pr=1151#step:7:15
```
The lock file might not be compatible with the current version of Poetry.
Upgrade Poetry to ensure the lock file is read properly or, alternatively, regenerate the lock file with the `poetry lock` command.
```

Regenerate the lock file with Poetry 1.8.2.